### PR TITLE
new change based on change in backend changes

### PR DESCRIPTION
**Pull request by anwara-ms**
The document had "ELC" written. We dont use ELCs anymore.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/retention-policies.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ The following are known issues for retention policies in Teams that are being tr
 
 - Under **Choose users** in the **Teams chats** location row, you may see guests and non-mailbox users. Retention policies aren't meant to be set for guests, and we're working to remove these from the list.
 
-- Exchange Life Cycle assistant (ELC) runs daily, but the latency is known to have run upto 7 days, in some cases. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
+- Retention processing jobs run daily, but the latency is known to have run upto 7 days, in some cases. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
```
Comment (JohanFreelancer9) [inline Teams/retention-policies.md:200]:
```suggestion
- Retention processing jobs run daily, but the latency is known to have run up to 7 days, in some cases. As a result, it's possible that, if you have a Teams retention policy to delete items older than 60 days, these items could persist for up to 67 days. This isn't a new situation - it follows the Exchange model. Of course, in most cases, there's no delay.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by scanum** [comment]
@cabailey PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cabailey** [comment]
Merging - however, I question why this is listed as a "known issue".  The up to 7 days is standard for retention policies and on this page covered previously under Considerations and limitations: "... it may take up-to seven days to permanently delete these messages from backend storage and Teams app"

**Comment by anwara-ms** [comment]
That’s to set expectation and something we are trying to improve on from backend services side.

We say if customer sets x day retention delete, messages should be deleted by Xth day, but there is a lag that customers don’t expect when setting policy and come back to us with a bug. We are saying that this is a known behavior or known issue.

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Pull request merged**